Syntax.in

```diff
@@ -0,0 +1,65 @@
+<program> ::= #<stmtList>#
+
+<stmtList> ::= <stmt> | <stmt><stmtList>
+
+<stmt> ::= <simpleStmt> | <structStmt> 
+
+<simpleStmt> ::= <declaration> | <iostmt> | <assignstmt> 
+
+<declaration> ::= var <identifier> :: <type> | var <identifier> :: <type> = <const>
+
+<const> ::= <char> | <int> | <str>
+
+<type> ::= <simpleType> | <arrayType>
+
+<simpleType> ::= char | int | str
+
+<arrayType> ::= arr[<simpleType> <unsignedNumber>]
+
+<iostmt> ::= read (<identifier>) | print (<identifierList>) | print (<const>)
+
+<identifierList> ::= <identifier> | <identifier>,<identifierList>
+
+<simpleAssignOp> ::= :=
+
+<specialAssignOp> ::= +:= | -:=
+
+<simpleAssign> ::= <identifier> <simpleAssignOp> <expression> 
+
+<specialAssign> ::= <identifier> <specialAssignOp> <const> 
```

Comment:
so you cannot have something like `x +:= y`
if that's on purpose it's fine. just mentioned in case you forgot

Reply:
it is on purpose